src/stubs/torch_bindings.ml

```diff
@@ -136,6 +136,13 @@ module C (F : Cstubs.FOREIGN) = struct
         (string
         @-> static_funptr Ctypes.(string @-> t @-> returning void)
         @-> returning void)
+
+    let loadz_callback =
+      foreign
+        "at_loadz_callback"
+        (string
+        @-> static_funptr Ctypes.(string @-> t @-> returning void)
+        @-> returning void)
   end
```

Comment:
The C++ bits are shared with the rust version of the bindings which support loading tensors directly to a device, this isn't supported on the OCaml side as it never was requested, I've still copied both C++ functions over just in case it becomes useful at some point.